jbake-core/src/main/java/org/jbake/launcher/Main.java

```diff
@@ -177,4 +176,12 @@ private void initStructure(CompositeConfiguration config, String type, String so
 		}
 	}
 
+	private void setupLocale(CompositeConfiguration config){
+		if (config.containsKey("jvm.language") && config.containsKey("jvm.country")){
+			String language = config.getString("jvm.language");
```

Comment:
Could you convert these config strings to constants like here: https://github.com/jbake-org/jbake/blob/master/jbake-core/src/main/java/org/jbake/app/ConfigUtil.java#L28